typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.stories.tsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Should try to honor prettier rules and only override in specific cases

Reply:
prettier is enabled, style fixed.